xbmc/windowing/egl/EGLNativeTypeIMX.cpp

```diff
@@ -338,7 +365,11 @@ bool CEGLNativeTypeIMX::ModeToResolution(std::string mode, RESOLUTION_INFO *res)
   res->iScreen       = 0;
   res->bFullScreen   = true;
   res->iSubtitles    = (int)(0.965 * res->iHeight);
-  res->fPixelRatio   = 1.0f;
+
+  // recalculated PR relatively to startup desktop resolution will result in 1.0f
+  // for all res with same AR and allows use of others without need to calibrate before
+  // by user.
+  res->fPixelRatio   = (float)m_init.fPixelRatio / (res->iScreenWidth/res->iScreenHeight);
```

Comment:
@koying 

here with the check for sanity
https://github.com/mk01/xbmc/commit/e1f401a18a13c5e7a563ee10512d93f6748ef792